programmers/k-th-number.js

```diff
@@ -0,0 +1,15 @@
+function solution(arry, commands){
+    const cd_len = commands.length;
+    let results = [];
+    for (let i = 0 ;  i < cd_len ; i++){
+        let start = commands[i][0]-1;
+        let end = commands[i][1];
+        let k = commands[i][2]-1;
+        let arryCuted = (arry.slice(start,end)).sort();
+        results.push(arryCuted[k]);
+    }
+    return console.log(results);
+}
+const arry = [1,5,2,6,3,7,4];
+const commands = [[2,5,3],[4,4,1],[1,7,3]]; 
+solution(arry,commands);
```

Comment:
가희님이 푸신 방식도 좋지만 조금 더 간결하게 코드를 짤 수 있습니다
```suggestion
function solution(array, commands) {
    let answer = []
    for (let i = 0; i < commands.length; i++) {
        const arr = array.slice(commands[i][0] - 1, commands[i][1]).sort((a, b) => a - b);
            answer.push(arr[commands[i][2] - 1])
    }
    return answer
}
```